Formula/libhttpserver.rb

```diff
@@ -3,8 +3,8 @@ class Libhttpserver < Formula
   homepage "https://github.com/etr/libhttpserver"
   url "https://github.com/etr/libhttpserver/archive/v0.9.0.tar.gz"
   sha256 "fbdc0a44e92e78e8cc03b0a595e6190d2de002610a6467dc32d703e9c5486189"
-  head "https://github.com/etr/libhttpserver.git"
   revision 1
+  head "https://github.com/etr/libhttpserver.git"
```

Comment:
There should be an extra blank line above the head line whenever there is a revision. If there is no revision, there should be no extra blank line above the head line. So in this case, line 7 should be a blank line and `head "https://github.com/etr/libhttpserver.git"` should be line 8.